### PR TITLE
FormSignupTestCase

### PR DESCRIPTION
**Pull request by paulgoujon**
Tests on the signup form integrated.

### CODE REVIEW

**Comment by paulgoujon** [comment]
please check if you think in any test I could have forgotten

**Comment by paulgoujon** [comment]
incoming : employeeform, paused while issue fixing

File: bagtrekkin/tests/test_signupForm.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        self.assertTrue(self.form.fields['first_name'].required)
+        self.assertTrue(self.form.fields['last_name'].required)
+        self.assertTrue(self.form.fields['email'].required)
+
+    def test_employee_is_created(self):
+        """Saving a form creates user AND employee LINKED to user"""
+        if self.form.is_valid():
+            self.form.save()
+        user = User.objects.get(username='totodu33')
+        self.assertTrue((user is not None))
```
Comment (roddehugo) [inline bagtrekkin/tests/test_signupForm.py]: self.assertIsNotNone(user)

File: bagtrekkin/tests/test_signupForm.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        employee = Employee.objects.get(user=user)
+        self.assertTrue((employee is not None))
```
Comment (roddehugo) [inline bagtrekkin/tests/test_signupForm.py]: self.assertIsNotNone(employee)

**Pull request merged**